internal/provider/resource_hvn_test.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 var (
-	hvn = fmt.Sprintf(`
+	testAccHvnConfig = fmt.Sprintf(`
```

Comment:
is this a convention we need to follow?

Reply:
No, I only changed this so that the name was more clear and less likely to be reused by other code in the package